.devcontainer/Dockerfile

```diff
@@ -0,0 +1,101 @@
+# Copyright 2020 The Crossplane Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# ====================================================================================
+
+FROM golang:1.13-buster
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+ARG USERNAME=vscode
+ARG USER_UID=1000
+ARG USER_GID=$USER_UID
+
+
+RUN apt-get update \
+    && apt-get -y install --no-install-recommends apt-utils dialog 2>&1 \
+    && apt-get -y install git iproute2 procps lsb-release \
+    # Make sure the packages below are installed, following specs from Crossplane's INSTALL.md
+    && apt-get install -yy -q --no-install-recommends \
+    build-essential \
+    ca-certificates \
+    curl \
+    docker.io \
+    jq \
+    make \
+    rsync \
+    runit \
+    unzip \
+    zip \
+    && mkdir -p /tmp/gotools \
+    && cd /tmp/gotools \
+    && GO111MODULE=on go get -v golang.org/x/tools/gopls@latest 2>&1 \
```

Comment:
what's your thoughts on using `latest` tags for all of these imports within the dev container? That means that depending on when a community members ends up using this for the first time, they will get a different set of versions for the tools, which makes this a less consistent and repeatable experience.  Do you think that will be an issue?

Reply:
I agree. We should stick to a version tag whenever is possible.